### DISCUSSION TITLE
Screenshots Wanted

### DISCUSSION THREAD

**Discussion by Taiko2k** [Taiko2k/Tauon#435] [General]
The current main featured screenshot in the readme is in need of replacement. Its a little old now, so would be nice to have a replacement and keep things fresh. It would be helpful if you could make and submit one! :camera_flash: 

Might also be nice to have more/different ones for the store pages. 

Alas making a great screenshot isn't easy though. Some guidelines:

 - For the main screenshot, try keep it a similar proportion/aspect-ratio as the current one.
 - Avoid imperfections, for example, don't show tracks with missing art or metadata
 - **[Update]** For screenshots for Flathub/GNOME Software, these must be exactly 16:9 ratio. ~~Achieving this is a little tricky. Perhaps I'll come up with an easy way of doing it in the future.~~ You can now use <kbd>Alt</kbd> + <kbd>F11</kbd> to force the window into a 16:9 ratio.

A little warning, remember, when a screenshot is featured, a lot of people will see it!

If you want to submit, just include it in a reply to this thread!

**Comment by Solarunit** [comment]
What about gifs? =)
I think that  a beautiful albums arts are also important.

**Reply by Taiko2k** [reply]
I think its better if the page loads fast, so GIF's are no good. Also the main screenshot should be still and not animated I think.

**Comment by Bauke** [comment]
![tauon-screenshot-celeste-(1326x811)](https://user-images.githubusercontent.com/19501722/104108719-2e148b80-52c7-11eb-92a1-8a1b48a0be1b.png)

![tauon-screenshot-rameses-b-(1326x811)](https://user-images.githubusercontent.com/19501722/104108722-3240a900-52c7-11eb-9bd7-876638247ac2.png)

Here are two screenshots that are (about) the same resolution as the one in the README, it's hard to get it exactly the same. :P

**Reply by Taiko2k** [reply]
It's okay, it needn't be exactly the same. Ive applied one of these now, thanks!

I'll leave this open if anyone wants to submit more/others. Perhaps I can rotate the featured screenshots occasionally.

**Reply by Solarunit** [reply]
Looks good, I would also add a view with left side panel...
BTW does github support carousel view?
Do you plan to replace screenshots on flathub?

**Reply by Taiko2k** [reply]
Yes I can replace the Flathub ones. I'll just need to pick and choose. I'll want the primary one to be a good one; one that is compact, since I think that's where Tauon looks the best.

Github doesn't support carousel. To keep the readme neat I limit the number of screenshots here.

**Reply by Solarunit** [reply]
>one that is compact, since I think that's where Tauon looks the best.

agreed

**Comment by msmafra** [comment]
![Captura de tela de 2021-01-11 13-31-51](https://user-images.githubusercontent.com/1005457/104210290-7344d480-5411-11eb-900f-21458cc8ff0a.png)
![Captura de tela de 2021-01-11 13-34-46](https://user-images.githubusercontent.com/1005457/104212185-b1db8e80-5413-11eb-88fe-2ccf30eed28c.png)
![Captura de tela de 2021-01-11 13-34-04](https://user-images.githubusercontent.com/1005457/104212190-b30cbb80-5413-11eb-9023-e2fd342dfd67.png)
![Captura de tela de 2021-01-11 13-33-13](https://user-images.githubusercontent.com/1005457/104212192-b3a55200-5413-11eb-869b-085b80df02fa.png)

**Reply by Taiko2k** [reply]
Looks good, but unfortunately the unloaded artist thumbnail is an imperfection.

**Reply by msmafra** [reply]
I did not notice it before, but all artists/albums are not loading the artist thumbnail. That's why I thought it was it.
The same albums on Pulsar (Android) are loading the respective thumbnails.
Could it be some Fedora/Flatpack restriction/permission?

**Reply by Taiko2k** [reply]
No, its last.FM's fault. They stopped allowing artist picture downloads.

**Comment by alan-null** [comment]
Hey @Taiko2k,
Is it ongoing thread? Do you still need screenshots.
I am happy to help with the project once I get my hands on it.

**Reply by Taiko2k** [reply]
Thanks for the offer but these aren’t really needed right now. I’ll close this discussion.

**Reply by C0rn3j** [reply]
You can however check out https://github.com/Taiko2k/Tauon/issues/1343 which is similar in nature
